workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/utils/calculateTrialTime.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {months} from './getMonths';
+
+export function calculateTrialTime() {
+	const date = new Date();
+
+	date.setTime(date.getTime() + 30 * 24 * 60 * 60 * 1000);
+
+	const endOfTrialDay = date.getDate();
+	const endOfTrialMonth = months[date.getMonth()];
+
+	return {
+		endOfTrialDay,
+		endOfTrialMonth,
+	};
+}
```

Comment:
Avoid using magic numbers, what `30 * 24 * 60 * 60 * 1000` means?

See this example: https://github.com/ryanmcdermott/clean-code-javascript?tab=readme-ov-file#use-searchable-names

```suggestion
import {months} from './getMonths';

CONST ASSIGN_A_BETTER_NAME_TO_ME = 30 * 24 * 60 * 60 * 1000;

export function calculateTrialTime() {
	const date = new Date();

	date.setTime(date.getTime() + ASSIGN_A_BETTER_NAME_TO_ME);

	const endOfTrialDay = date.getDate();
	const endOfTrialMonth = months[date.getMonth()];

	return {
		endOfTrialDay,
		endOfTrialMonth,
	};
}
```

Reply:
Hi @kevenleone this was not developed by me, I just replaced this function in this file, but will be fixed!